com.unity.uiwidgets/Runtime/foundation/debug.cs

```diff
@@ -56,17 +60,37 @@ public static class D {
 
         [Conditional("UNITY_ASSERTIONS")]
         public static void assert(Func<bool> result, Func<string> message = null) {
-            if ( UIWidgetsPanel.ShowDebugLog && !result() ) {
+            if ( _enableDebug && !result() ) {
                 throw new AssertionError(message != null ? message() : "");
             }
         }
 
         [Conditional("UNITY_ASSERTIONS")]
         public static void assert(bool result, Func<string> message = null) {
-            if (UIWidgetsPanel.ShowDebugLog && !result  ) {
+            if ( _enableDebug && !result  ) {
                 throw new AssertionError(message != null ? message() : "");
             }
-        } 
+        }
+
+        public static bool _enableDebug {
+            get {
+                return EditorPrefs.GetBool("EnableDebugLog");
```

Comment:
we need store the enableDebug locally in this class. calling EditorPrefs.GetBool frequently will slow down the framework